src/main/resources/view/PersonListCard.fxml

```diff
@@ -20,24 +20,24 @@
       <padding>
         <Insets top="5" right="5" bottom="5" left="15" />
       </padding>
-      <HBox spacing="0.5" alignment="CENTER_LEFT">
-        <Label fx:id="id" styleClass="cell_big_label">
+      <HBox spacing="0.5" alignment="TOP_LEFT">
+        <Label fx:id="id" styleClass="cell_big_label" >
           <minWidth>
             <!-- Ensures that the label text is never truncated -->
             <Region fx:constant="USE_PREF_SIZE" />
           </minWidth>
         </Label>
-        <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
+        <Label fx:id="name" text="\$first" styleClass="cell_big_label" wrapText="true"/>
       </HBox>
       <VBox spacing="4">
         <FlowPane fx:id="tags" />
         <FlowPane fx:id="appointments" />
       </VBox>
-      <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
-      <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
-      <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
-      <Label fx:id="age" styleClass="cell_small_label" text="\$age" />
-      <Label fx:id="sex" styleClass="cell_small_label" text="\$sex" />
+      <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" wrapText="true"/>
+      <Label fx:id="address" styleClass="cell_small_label" text="\$address" wrapText="true"/>
+      <Label fx:id="email" styleClass="cell_small_label" text="\$email" wrapText="true"/>
+      <Label fx:id="age" styleClass="cell_small_label" text="\$age" wrapText="true"/>
```

Comment:
Not sure how it looks like after the changes, perhaps it would be clearer if we include screenshots in pr description~ but LGTM! Thank you!

Reply:
Oh I was thinking it's a bit self-explanatory due to the text wrap but basically the texts will just continue on the next line if it's too long. I've include the screenshot of it now. Thanks for the suggestion!